hydromt_wflow/workflows/soilparams.py

```diff
@@ -155,35 +156,37 @@ def update_kvfrac(
             kv_depth = calc_kv_at_depth(depth=depth, kv_0=kv0_mask, f=f_mask)
             paddy_values = calc_kvfrac(kv_depth=kv_depth, target=target)
             # Set the values in the correct places
-            kvfrac = xr.where(
+            soil_ksat_vertical_factor = xr.where(
                 paddy_values.raster.mask_nodata().isnull(),
                 da_kvfrac.loc[dict(layer=idx)],
                 paddy_values,
             )
-            kvfrac = kvfrac.fillna(-9999)
-            kvfrac.raster.set_nodata(-9999)
+            soil_ksat_vertical_factor = soil_ksat_vertical_factor.fillna(-9999)
+            soil_ksat_vertical_factor.raster.set_nodata(-9999)
             # Update layer in dataarray
-            da_kvfrac.loc[dict(layer=idx)] = kvfrac
+            da_kvfrac.loc[dict(layer=idx)] = soil_ksat_vertical_factor
 
     return da_kvfrac
 
 
-def get_ks_veg(KsatVer, sndppt, LAI, alfa=4.5, beta=5):
+def get_ks_veg(
+    soil_ksat_vertical, sndppt, vegetation_leaf_area_index, alfa=4.5, beta=5
```

Comment:
+ update below accordingly
```suggestion
    ksat_vertical, sndppt, LAI, alfa=4.5, beta=5
```